src/ucp/api/ucp.h

```diff
@@ -325,6 +325,8 @@ typedef struct ucp_config {
     size_t                                 bcopy_thresh;
     /** Threshold for switching UCP to rendezvous protocol */
     size_t                                 rndv_thresh;
+    /** How much packet data to dump in debug mode */
```

Comment:
/** The size (in bytes) of the log buffer that is used for a logging of the network packet information (headers, payload, etc.). The information that does not fit into the log buffer will be truncated.*/